### ISSUE TITLE
Example doesn't work (can't plot forecast)

### ISSUE THREAD

**Issue by giannilmbd** [bsvars/bsvars#80]
Hi,
Thanks a lot for making your conde available.
I've tried to run the example you provide and the code
```
out |> forecast(horizon = 8) |> plot()        # compute and plot forecasts
```
returns an error
```
Error in xy.coords(x, y, xlabel, ylabel, log) : 
  'x' is a list, but does not have components 'x' and 'y'
```
Indeed forecast(out,horizon=8) is a list with two elements: forecast and forecast_sigma (#variables,#horizon,#draws?)

Thanks

Gianni

**Comment by giannilmbd** [comment]
Thanks Tomasz. This is what I ran (from the main page of this github site)
```
library(bsvars)                               # upload the package
data(us_fiscal_lsuw)                          # upload data
us_fiscal_lsuw |>
  specify_bsvar_sv$new(p = 4) |>              # specify the model
  estimate(S = 1000) |>                       # run the burn-in
  estimate(S = 50000) -> out                  # estimate the model

out |> forecast(horizon = 8) |> plot()        # compute and plot forecasts
out |> compute_impulse_responses(8) |> plot() # compute and plot impulse responses
```
BSVARS verssion ‘2.1.0’
R version 4.2.1

Thanks
Gianni

**Comment by donotdespair** [comment]
OK, all clear now! 

The estimation and plotting code comes from the newest developers version 2.1.0.9000. And you are using the currently available version on CRAN 2.1.0. What you can do is:
1. install the developers version of the package using `devtools::install_github("bsvars/bsvars")`
2. Or wait a few days for the version 3.0 to land on the CRAN repository (I have already submitted it and await accepatnce). In such a case, you will use code `install.packages("bsvars")` to update the package.

Does this help?

Greetings, Tomasz

**Comment by giannilmbd** [comment]
Excellent. Thanks, I'll do as you suggest.
Cheers
Gianni
